argilla/docs/how_to_guides/annotate.md

```diff
@@ -0,0 +1,195 @@
+
+![UI overview](../assets/images/how_to_guides/annotate/ui_overview.png)
+
+!!! tip
+    The app is responsive which enable you to adapt your workspace from two to one column. You can even use your mobile to provides simple feedback on your datasets.
+
+### Shortcuts
+
+Argilla UI includes a range of shortcuts. For the main actions submit, discard, save as draft and the labels the keys are showed in the button.
+
+To move from a question to another, or between records using the keyboard you may have a look to the table just below.
+
+!!! tip
+    Shortcuts provide a smoother experience, especially with a long list of labels or single-question forms.
+
+??? "Consult the available shortcuts"
+
+    | Action | Keys |
+    | --- | --- |
+    | Activate form | ⇥ Tab |
+    | Move between questions | ↓ Down arrow or ↑ Up arrow |
+    | Select and unselect label | 1, 2, 3 |
+    | Move between labels or ranking options | ⇥ Tab or ⇧ Shift ⇥ Tab |
+    | Select rating and rank | 1, 2, 3 |
+    | Fit span to character selection | Hold ⇧ Shift |
+    | Activate text area | ⇧ Shift ↵ Enter |
+    | Exit text area | Esc |
+    | Discard | ⌫ Backspace |
+    | Save draft (Mac os) | ⌘ Cmd S |
+    | Save draft (Other) | Ctrl S |
+    | Submit | ↵ Enter |
+    | Move between pages | → Right arrow or ← Left arrow |
+
+### View by status
+
+The view selector is set by default on Pending.
+
+If you are starting an annotation effort, all the records are initially kept in the Pending view. Once you annotate little by little, the records will move to the other queues: Draft, Submitted, Discarded.
+
+- **Pending**: The records with no responses.
+- **Draft**: The records with partial responses. They can be submitted or discarded later. They can’t go back to the pending queue.
+- **Discarded**: The records may or may not have responses. They can be edited but can’t go back to the pending queue.
+- **Submitted**: The records have been fully annotated and have already been submitted.
+
+### Suggestions
+
+If your dataset includes model prediction, you will see them represented by a sparkle icon `✨` in the label or value button. We call them “Suggestions”, and they appear in the form as pre-filled responses. If the suggestions convince you, you just need to click on the Submit button, and they will be considered as another annotation.
+
+If the suggestion is incorrect, you can modify it and submit it.
+
+The score per suggested label/value is displayed for MultiLabelQuestion and RankingQuestion.
+You can also order your labels in the preference settings in order to show suggestions first, on top of the label list.
+
+### Focus view
+
+![Focus view](../assets/images/how_to_guides/annotate/focus_view.png)
+
+This is the default view to annotate your dataset linearly, displaying one record after another.
+
+**You should use this when:** getting acquainted with a dataset or when the annotation team is very diverse, the topic is generic.
```

Comment:
@sdiazlor can we turn this into tips to make it more standardized? After that, we'll add some edits to the content itself, ping us on slack once these are turned into tips